webpack/dev-server.js

```diff
@@ -61,6 +61,9 @@ app.use(hotMiddleware);
 
 // serve pure static assets
 const staticPath = path.posix.join(config.dev.assetsPublicPath, config.dev.assetsSubDirectory);
+app.use('/static/icons', express.static(path.posix.join(config.dev.staticAssetsDirectory, 'icons/'), {
```

Comment:
This file is only used for local testing and development, in production the static files are served from here https://github.com/HabitRPG/habitica/blob/develop/website/server/middlewares/static.js

Reply:
@paglias Thanks - I'm still getting familiar with the codebase  :)  Is there a way I can test this locally?

Reply:
To test this locally you should run the codebase with the env var `NODE_ENV` set to `production` but that requires some extra changes that I'm not sure are possible locally...

Anyway the change in the other file looks good

Reply:
This change should be removed, we don't want caching while testing locally :)